src/statsd_server.rs

```diff
@@ -110,10 +113,11 @@ fn process_buffer_newlines(buf: &mut BytesMut) -> Vec<StatsdPDU> {
     return ret;
 }
 
-async fn client_handler(
+async fn client_handler<T: AsyncRead + AsyncWrite + Send + Sync + Unpin + 'static>(
```

Comment:
That's a lot of bounds — given it's tokio I think I already know the answer, but is there a trait we can use to express "some kind of SOCK_STREAM" more succinctly?

Reply:
Simplified the bounds a bit and moved it to a where clause for good measure